tests/Rector/PropertyFetch/OptionalToNullsafeOperatorRector/config/configured_rule.php

```diff
@@ -9,12 +9,10 @@
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/../../../../../config/config.php');
 
-    $services = $rectorConfig->services();
-
-    $services->set(OptionalToNullsafeOperatorRector::class)
-        ->call('configure', [
-            [
-                OptionalToNullsafeOperatorRector::EXCLUDE_METHODS => ['present'],
-            ],
-        ]);
```

Comment:
@TomasVotruba I tried using `UpgradeRectorConfigRector`, it seems when using old `->call('configure')`, it replace incorrectly, I changed it manually to use direct array values.